mungegithub/mungers/sig-mention-handler_test.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mungers
+
+import (
+	"reflect"
+	"runtime"
+	"testing"
+
+	githubapi "github.com/google/go-github/github"
+	github "k8s.io/test-infra/mungegithub/github"
+)
+
+const (
+	commentMessage = "There are no @kubernetes/sig mention on the issue. Please mention one."
```

Comment:
Use the same comment from the sig-mention-handlers.go file.  No need to make another one